### DISCUSSION TITLE
关于LAB2的insert的redo 插入有一些疑问？

### DISCUSSION THREAD

**Discussion by AshenskyABCDE** [thu-db/huadb#13] [Q&A]
我在实现Insert的redo的时候遇到了一个疑问   
实现`void TablePage::RedoInsertRecord(slotid_t slot_id, char *raw_record, db_size_t page_offset, db_size_t record_size)`这里，文档提示说 将 raw_record 写入 page data。
这里是通过序列化的方式写入吗？ 如果要用到 `DeserializeFrom(record_data, column_list)` 的话table_page里面是没有column_list的，所以在这个方面很困扰。table_page的TablePage::RedoInsertRecord 的 将 raw_record 写入 page data 是用何种方法？

**Comment by hnjylwb** [comment]
raw_record 本身已经为 char * 类型，直接使用 memcpy 函数写入即可。
